src/content/commandline_content.ts

```diff
@@ -39,6 +39,24 @@ async function init() {
         hide()
         document.documentElement.appendChild(cmdline_iframe)
         enabled = true
+
+        // Determine the initial page zoom level when the it first loads
+        // So that we can use this value later
+        const zoomLevelOnPageLoad = Math.round(window.devicePixelRatio * 100)
```

Comment:
This introduces a bug:

1. zoom in on a page
2. reload / visit in a new tab
3. tridactyl is now zoomed in